src/frontend/src/icp-eth/components/tokens/ManageTokens.svelte

```diff
@@ -3,6 +3,9 @@
 	import { debounce, nonNullish } from '@dfinity/utils';
 	import { createEventDispatcher, onMount } from 'svelte';
 	import { fade } from 'svelte/transition';
+	import BtcManageTokenToggle from '$btc/components/BtcManageTokenToggle.svelte';
```

Comment:
I'm not sure anything finding place in `$icp-eth` should reference `$btc` or we should rename the package.

I tend to think that we need a new package or this should find place in `lib`. Not sure what's best.

Reply:
That's a good point, I also saw some other files within `$icp-eth` that seem to belong more to `lib`.

I believe that the idea behind `icp-eth` was about what ICP and ETH share because those where the only two networks. But with Bitcoin, most of the stuff here is susceptible to be reused.

Maybe rename to `icp-eth-btc`? Or `shared`? Or move everything to `lib`. I don't think there will be many things that are only ICP or ETH...

Reply:
Let's move it to `$lib`, at least for now. https://github.com/dfinity/oisy-wallet/pull/2332